dts/bindings/timer/st,stm32-lptim.yaml

```diff
@@ -10,3 +10,32 @@ include:
     property-blocklist:
         - st,prescaler
         - st,countermode
+
+properties:
+    st,prescaler:
+      type: int
+      required: false
+      default: 1
+      description: |
+        Clock divider at the input of the lptimer clock, default reset value is 'not divided'.
+
+        Prescaler allows to achieve higher LPTIM timeout (up to 256s when lptim clocked by LSE)
+        and consequently higher core sleep durations, but impacts the tick precision.
+        To be used with caution.
+        CONFIG_SYS_CLOCK_TICKS_PER_SEC could be used to tune tick duration and gain precision,
```

Comment:
Do you really need this? I know there's not a great place to put driver specific information in our documentation, but this seems like a clear layering violation. DT bindings for an individual SoC timer should be pretty far away from zephyr system clock configuration in the layer, right?

Reply:
>  DT bindings for an individual SoC timer should be pretty far away from zephyr system clock configuration in the layer, right?

This is not system clock configuration, but systick rate, which is directly linked because this timer is the actual tick source when PM is used (on the whole stm32 family).

Question is then where to put this information because this prescaler has direct consequences on tick precision (to gain sleep duration), which might need to be counter balanced by a fine tuning of `CONFIG_SYS_CLOCK_TICKS_PER_SEC`.